_topics/en/channel-jamming-attacks.md

```diff
@@ -0,0 +1,108 @@
+---
+title: Channel jamming attacks
+
+## Optional.  Shorter name to use for reference style links e.g., "foo"
+## will allow using the link [topic foo][].  Not case sensitive
+# shortname: foo
+
+## Optional.  An entry will be added to the topics index for each alias
+#aliases:
+#  - Foo
+
+## Required.  At least one category to which this topic belongs.  See
+## schema for options
+categories:
+  - Lightning Network
+
+## Required.  Use Markdown formatting.  Only one paragraph.  No links allowed.
+## Should be less than 500 characters
+excerpt: >
+  **Channel jamming attacks** are Denial of Service (DoS) attacks where
+  an attacker can prevent a series of channels up to 20 hops away from
+  being able to use part or all of their funds for a prolonged period of
+  time.
+
+## Optional.  Use Markdown formatting.  Multiple paragraphs.  Links allowed.
+extended_summary: |
+  An LN node can route a payment to itself across a path of 20 or more
+  hops. This creates two possible avenues for channel jamming attacks:
+
+  - *Liquidity jamming attack* (originally called the [loop attack][russell
+    loop] in 2015) is where an attacker with 1 BTC (for example) sends
```

Comment:
I think it's possibly a bit confusing to lead with the specific "1 BTC" amount rather than make it more generic and then giving an example.